src/template.html

```diff
@@ -8,21 +8,24 @@
 <body>
   <main>
 
-    <h1>Leaderboard</h1>
+    <div class="title">
+      <h1>Leaderboard</h1>
+      <div class="msg-container"></div>
+    </div>
     <div class="section-container">
       <section class="left-block">
         <div class="header-lb">
           <h2>Recent scores</h2>
-          <button class="btn">Refresh</button>
+          <button class="btn refresh-btn">Refresh</button>
         </div>
-        <div class="score-container"></div>
+        <ul class="score-container"></ul>
       </section>
 
       <section class="right-block">
         <h2>Add your score</h2>
         <form action="index_submit" method="post" accept-charset="utf-8">
-          <input type="text" value="" name="name"/>
-          <input type="number" value="" name="score"/>
+          <input type="text" value="" name="name" required maxlength="20"/>
+          <input type="number" value="" name="score" required min="0" max="1000000"/>
```

Comment:
- [ ] **[OPTIONAL]** Good job with this awesome project 💯. However, I suggest you add placeholder text to these inputs so that users know exactly the input that is for name and the one for number.

Reply:
### Hi @julie-ify :wave: 
Thanks for your feedback, I will implement the suggestions.